app/Http/Requests/MeetingRequest.php

```diff
@@ -109,23 +114,33 @@ public function attributes(): array
             'end_time' => __('meeting end time'),
             'date' => __('meeting date'),
             'timezone' => __('meeting time zone'),
+            'locality' => __('city or town'),
             'region' => __('province or territory'),
             'meeting_url' => __('link to join the meeting'),
-            'meeting_phone' => __('meeting phone number'),
+            'meeting_phone' => __('phone number to join the meeting'),
```

Comment:
@greatislander do you think something like this would still fit into the validation error message after being translated into French?

Reply:
Yes, I think so.